picard/track.py

```diff
@@ -71,8 +71,23 @@ def add_file(self, file):
     def update_file_metadata(self, file):
         if file not in self.linked_files:
             return
-        file.copy_metadata(self.metadata)
+        file.copy_metadata(self.orig_metadata)
         file.metadata['~extension'] = file.orig_metadata['~extension']
+        self.metadata.copy(file.metadata)
+
+        # Re-run tagger scripts with updated metadata
+        if config.setting["enable_tagger_scripts"]:
+            for s_pos, s_name, s_enabled, s_text in config.setting["list_of_scripts"]:
+                if s_enabled and s_text:
+                    parser = ScriptParser()
+                    try:
+                        parser.eval(s_text, file.metadata)
+                        parser.eval(s_text, self.metadata)
+                    except:
+                        log.error(traceback.format_exc())
```

Comment:
Since we moved to logging module you should be able to use `log.exception()` in place of `log.error(traceback.format_exc())`
https://github.com/metabrainz/picard/blob/master/picard/log.py#L137
https://docs.python.org/3/library/logging.html#logging.exception